src/main/grammars/LalrpopLexer.flex

```diff
@@ -47,7 +47,7 @@ StrLiteral = \" ([^\\\"] | \\. )* \"
 
 Path = (::)? {Id} (:: {Id})* (::\*)?
 Id = [a-zA-Z][a-zA-Z0-9_]*
-Lifetime = \' {Id}
+Lifetime = \' {Id} | \' _
```

Comment:
I think we might want to adjust the definition of `Id` to allow leading underscores. Then we don't need the special case in `Lifetime`.